test/Constraints/operator.swift

```diff
@@ -284,7 +284,8 @@ func rdar60727310() {
 // FIXME: Bad diagnostic.
```

Comment:
Nit: should we update the fixme? I was about to comment for removing the fixme, but although fallback is good here, a "no exact matches for operator" with candidate notes feels would fit better in this case.